edk2toollib/windows/policy/firmware_policy.py

```diff
@@ -0,0 +1,561 @@
+# @file
+#
+# Library to parse/manage/build unsigned firmware policy blobs
+#
+# Copyright (c), Microsoft Corporation
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+
+##
+# A firmware policy is conceptually a set of key value pair "rules".
+# Rules can be 1-time actions, states persistent while the policy is in effect,
+# or targeting information describing the machine where the policy is applicable
+#
+# Each key is composed of a UINT32 RootKey followed by String SubKeyName & ValueName
+# For example:
+#   RootKey = 0xEF100000
+#   SubKeyName = "PolicyGroupFoo"
+#   ValueName = "TpmClear"
+#   MyExampleKey = EF100000\PolicyGroupFoo\TpmClear
+#
+# Each value has a type that can be primitive or a variable-length structure.
+#
+##
+# The policy binary, pack(1), little endian
```

Comment:
Can you link to where each of these structures is defined in C?

Reply:
You're looking at the egg, the 🐔 will hatch in another repo soon.  At that time, great point.  Where do we keep track of TODOs, Issue Tracker?

Reply:
I took a note to self to add this later when the C struct is posted